ajax/libs/xterm/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "xterm",
+  "filename": "xterm.js",
```

Comment:
We should include/generate `.min` files, so please change filename to `xterm.min.js`

Our recommend minify tool is this one: https://github.com/PeterDaveHello/web-minify-helper